src/main/java/gradle_clojure/clojure/tasks/ReflectionWarnings.java

```diff
@@ -26,6 +28,7 @@ public ReflectionWarnings(boolean enabled, boolean projectOnly, boolean asErrors
     this.asErrors = asErrors;
   }
 
+  @Input
```

Comment:
This might be more of an `@Console` as it doesn't really change the behavior of the build itself.

Reply:
I am not sure about this one @ajoberstar. `enabled` along with `asErrors` will change the result of the build as it controls not only printing them to the console but also if the build should be failed.

Reply:
I was thinking that if `asErrors` stays as `@Input`  it would still cover that scenario. If it wouldn't then I'd agree with leaving enabled that way too.

Reply:
I have checked that the following case is not handled correctly with `enabled` marked as `@Console` in a project that produces reflection warnings during compilation:

```
reflectionWarnings {
  enabled = false
  asErrors = true
}
```

Run `./gradlew check` - build succeeds.

Then change `enabled` to true and run `./gradlew check` - `testClojure` is marked as up to date whereas with `enabled` marked as `@Input` causes `compileClojure` to be rerun with a build failure.

Thus I think `enabled` should be annotated as `@Input` as [it influences not only console output but also the build status](https://github.com/gradle-clojure/gradle-clojure/blob/86b5d813bf4497ead737f75fb8499a6d26b3a7d1/src/main/java/gradle_clojure/clojure/tasks/ClojureCompile.java#L170-L172) (depending on the value of `asErrors`).

Reply:
OK, sounds good to me.